packages/console/src/components/Switch/index.tsx

```diff
@@ -3,16 +3,16 @@ import { forwardRef } from 'react';
 
 import * as styles from './index.module.scss';
 
-type Props = Omit<HTMLProps<HTMLInputElement>, 'value'> & {
+type Props = HTMLProps<HTMLInputElement> & {
```

Comment:
would you like to add an issue for adding unit tests for fundamental components

Reply:
I've created a project to track this.
https://linear.app/silverhand/project/unit-tests-for-fundamental-components-in-admin-console-79af882de43b